src/tracker/encoder/os_net/osnet.py

```diff
@@ -427,11 +429,12 @@ def init_pretrained_weights(model, key=""):
 
     Layers that don't match with pretrained layers in name or size are kept unchanged.
     """
-    import os
     import errno
-    import gdown
+    import os
     from collections import OrderedDict
 
+    import gdown
```

Comment:
I'm surprised you're importing things inside a function, instead of at the top of the file. What happens if this function is run twice? Mild preference to put all imports at the top, so I don't need to ask that question.

(Rarely, importing in a function is important. If that's the case, put in a comment explaining why, such as "the server connection gets initialized on import, and will time out if we haven't used it in the past hour. So, only import it when we're about to use it.")

Reply:
this part of a code is from this [code](https://github.com/KaiyangZhou) that is under MIT License, I just added the license. I am not using this because weights are part of the PyInstaller exec